storybookjs/storybook
# UI: Update mono font stack

Issue: #16278 

## What I did

1. I updated the monospace font stack to be a little more consistent across browsers and operating systems.
2. I updated the `Basics/Typography` story to better represent the typefaces.

**TODO**

<del>I would like to update [the story for Storybook's typography](https://github.com/storybookjs/storybook/blob/3909ed6ad78291228707cdc41d7a2cdd73c17c1f/lib/components/src/typography/typography.stories.tsx) to use the `Typeset` docblock, but I wasn't able to do it myself. I could use an assist from @yannbf or @kylegach if either of you have some time to spare over the next few days.</del>

Update: Thanks, Kyle G. for the assist

## How to test

1. Open up Storybook Official via the link below
2. Try out the Actions addon or the Interactions addon to see examples of components using the monospace font.

### Before
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/1123119/138003204-f73df674-aa09-44dd-9b05-62f3a317bb64.png">
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/1123119/138003125-65ae6f44-245b-404c-bee7-1ef5be87ceef.png">

### After
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/1123119/138003249-c7a17fa3-e221-4f8a-bce8-47a0a64695f1.png">
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/1123119/138002985-bfd3b059-04d5-4916-92d5-fb01c7cc51ee.png">

<hr>
